code-challenges.test.js

```diff
@@ -43,12 +79,38 @@ var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 
 // b) Create the function that makes the test pass.
 
-
+// I will need an array to store my output, which I will initialize at the beginning as empty.
+// I can loop through the input with a for loop, and use the typeof operator to check that my index contains a number
+// if the current index is a number, then I push it to my storage array
+// after the loop, I can use the sort function and then return
+// I'm going to try to refrain from looking up the syntax, so for posterity here is what I think it is: array.sort(a, b => a-b)
+// fingers crossed
+
+const oddsInOrder = data => {
+  let numbers = []
+  for(let i=0; i<data.length; i++){
+    if(typeof data[i] == 'number' && data[i]%2 != 0){ //I forgot to factor in odds only in my pseudocode, but a simple fix in our conditional will solve it
+      numbers.push(data[i])
+    }
+  }
+  numbers.sort((a, b) => a-b)
+  return numbers
+}
```

Comment:
Excellent, and yea in JS if you have more than one param you'll need parens... (95% certainty)